google-youtube.html

```diff
@@ -647,6 +747,9 @@
       this.currenttime = Math.round(this._player.getCurrentTime());
       this.currenttimeformatted = this._toHHMMSS(this.currenttime);
       this.fractionloaded = this._player.getVideoLoadedFraction();
+      this.volume = this._player.getVolume();
+      this.playbackrate = this._player.getPlaybackRate();
```

Comment:
You're right, but regarding the volume do you think it would be better to decrease the interval in `this.__updatePlaybackStatsInterval = setInterval(this._updatePlaybackStats.bind(this), 1000);` or maybe create a separate thread only for the volume? Right now it's a little slow on the update process.